docs/audit-log-filter-variables.md

```diff
@@ -0,0 +1,772 @@
+None.
+
+#### Returns
+
+This function returns a string of a JSON array of the audit events, or a JSON NULL value. Returns `NULL` and generates an error if the call fails.
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT audit_log_read_bookmark();
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +----------------------------------------------------+
+    | audit_log_read_bookmark()                          |
+    +----------------------------------------------------+
+    | {"timestamp" : "2023-06-02 09:43:25", "id": 10 }   |
+    +----------------------------------------------------+
+    ```
+
+### audit_log_filter_remove_filter(filter_name)
+
+This function removes the selected filter from the current set of filters.
+
+If user accounts are assigned the selected filter, the user accounts are no longer filtered. The user accounts are removed from `audit_log_filter_user`. If the user accounts are in a current session, they are detached from the selected filter and no longer logged.
+
+#### Parameters
+
+`filter_name` - a selected filter name as a string.
+
+
+#### Returns
+
+This function returns either an `OK` for success or an error message for failure.
+
+If the filter name does not exist, no error is generated. 
+
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT audit_log_filter_remove_filter('filter-name');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +------------------------------------------------+
+    | audit_log_filter_remove_filter('filter-name')  |
+    +------------------------------------------------+
+    | OK                                             |
+    +------------------------------------------------+
+    ```
+
+### audit_log_filter_remove_user(user_name)
+
+This function removes the assignment of a filter from the selected user account.
+
+If the user account is in a current session, they are not affected. New sessions for this user account use the default account filter or are not logged.
+
+If the user-name is `%`, the default account filter is removed.
+
+#### Parameters
+
+`user_name` - a selected user name in either the `user_name`@`host_name` format or `%`.
+
+
+#### Returns
+
+This function returns either an `OK` for success or an error message for failure.
+
+If the user_name has no filter assigned, no error is generated. 
+
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT audit_log_filter_remove_user('user-name@localhost');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +------------------------------------------------------+
+    | audit_log_filter_remove_user('user-name@localhost')  |
+    +------------------------------------------------------+
+    | OK                                                   |
+    +------------------------------------------------------+
+    ```
+
+### audit_log_rotate()
+
+This function updates the audit log filter tables and makes any changes operational. 
```

Comment:
Wrong description here. Should be https://dev.mysql.com/doc/refman/8.0/en/audit-log-reference.html#function_audit-log-rotate